pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -136,6 +136,11 @@ type ActualStateOfWorld interface {
 	// have no mountedPods. This list can be used to determine which volumes are
 	// no longer referenced and may be globally unmounted and detached.
 	GetUnmountedVolumes() []AttachedVolume
+
+	// GetPodList generates and returns a map of pods in which map is indexed
+	// with pod's unique name. This map can be used to determine which pod is currently
+	// in actual state of world.
+	GetPodList() map[volumetypes.UniquePodName]bool
```

Comment:
nit: I'm probably being pedantic here. But you're calling this method `...List()` and it returns a map. So maybe just `GetPods()`?